core/tests/protractor/accessibility.js

```diff
@@ -101,9 +109,10 @@ describe('screenreader and keyboard user accessibility features', function() {
 
   it('should skip to the main content element', async function() {
     await libraryPage.get();
-    await browser.actions().sendKeys(protractor.Key.TAB).perform();
+    await action.sendKeys(
+      'Browser', browser, protractor.Key.TAB).perform();
     await waitFor.elementToBeClickable(skipLink, 'Could not click skip link');
```

Comment:
Now that you're using `actions` below, this line isn't needed. `actions.click` includes a `waitFor.elementToBeClickable`

Reply:
Done